src/main/java/sorald/FileUtils.java

```diff
@@ -114,6 +117,27 @@ public static void writeStatisticsJSON(
                 StandardOpenOption.TRUNCATE_EXISTING);
     }
 
+    /**
+     * Write the statistics JSON file.
+     *
+     * @param statsOutputFile The file to write to.
+     * @param statsCollector A {@link SoraldEventHandler} containing stats.
+     * @param executionInfo The information of the execution.
+     * @throws IOException If the file can't be written to.
+     */
+    public static void writeStatisticsJSON(
```

Comment:
@slarse  Do you think we can merge this function and the old `writeStatisticsJSON` (which was used for repair mode)? The old one receives a `StatisticsCollector` instead of `SoraldEventHandler`.

Reply:
@khaes-kth It could just receive `Object`, really. The method itself doesn't need to know anything about the object it receives.

Reply:
@khaes-kth  I'll open up a new PR with a more generalized version of the `writeStatisticsJSON` method, such that we can both have it.